include/mbedtls/x509_crt.h

```diff
@@ -346,8 +395,37 @@ int mbedtls_x509_crt_parse_file( mbedtls_x509_crt *chain, const char *path );
  *                 if partly successful or a specific X509 or PEM error code
  */
 int mbedtls_x509_crt_parse_path( mbedtls_x509_crt *chain, const char *path );
-#endif /* MBEDTLS_FS_IO */
 
+#endif /* MBEDTLS_FS_IO */
+/**
+ * \brief          Parses a subject alternative name item
+ *                 to an identified structure;
```

Comment:
Minor: `;` -> `.`
Minor: I find the wording "identified structure" odd. Can't we say "This function parses an item in the SubjectAlternativeNames extension."?